src/Paket.Core/Cultures.fs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Paket
+open System
+open System.Collections.Generic
+open System.Globalization
+
+[<CompilationRepresentation (CompilationRepresentationFlags.ModuleSuffix)>]
```

Comment:
Unless you have a type `Cultures` under `Paket` namespace, I don't think we need those attributes (which is used to disambiguate between module and type bearing same name).